### DISCUSSION TITLE
DocumentCodeType 623 Speditonsrechnung?

### DISCUSSION THREAD

**Discussion by kane95443** [stephanstapel/ZUGFeRD-csharp#451] [Q&A] [answered]
Hallo,
can`t find the Invoice Type (DocumentCodeType) '623' Speditionsrechnung. Is there a way for a hobby programmer to add this type or is there an encrypted .dll? Sorry if the question seems very simple for developers.

**Comment by stephanstapel** [comment, marked as answer]
I can quickly do this for you. You will need to use the Github code for the next week until I provide a new version of the package on nuget

**Reply by kane95443** [reply]
Thank you :-)

**Reply by stephanstapel** [reply]
done

**Reply by kane95443** [reply]
thank you for your quick reply and your help

**Comment by kane95443** [comment]
Hallo Stephan,

maybe we need 532 "Gutschrift des Spediteurs" too.

maybe you can take this codes from CEN/TS16931-3-3 (Profil Minimum) guideline:
Document ZUGFeRD 2.3.2 / Factur-X 1.07.2 Copyright AWV e.V., FNFE 2012-2024 from 13.11.2024

Code
Codename
.
71 . Request for payment
.
80 . Debit note related to goods or services
.
81 . Credit note related to goods or services
.
82 . Metered services invoice
.
83 . Credit note related to financial adjustments
.
84 . Debit note related to financial adjustments
.
102 . Tax notification
.
130 . Rechnungsdatenblatt
.
202 . Verkürzte Baurechnung
.
203 . Vorläufige Baurechnung
.
204 . Baurechnung
.
211 . Zwischen-(abschlags-)rechnung
.
218 . Final payment request based on completion of work
.
219 . Payment request for completed units
.
261 . Self billed credit note
.
262 . Consolidated credit note - goods and services
.
295 . Price variation invoice
.
296 . Credit note for price variation
.
308 . Delcredere credit note
.
325 . Proformarechnung
.
326 . Teilrechnung
.
331 . Commercial invoice which includes a packing list
.
380 . Handelsrechnung
.
381 . Gutschriftanzeige
.
382 . Provisionsmitteilung
.
383 . Belastungsanzeige
.
384 . Rechnungskorrektur
.
385 . Konsolidierte Rechnung
.
386 . Vorauszahlungsrechnung
.
387 . Mietrechnung
.
388 . Steuerrechnung
.
389 . Selbst ausgestellte Rechnung (Gutschrift im Gutschriftsverfahren)
.
390 . Delkredere-Rechnung
.
393 . Inkasso-Rechnung
.
394 . Leasing-Rechnung
.
395 . Konsignationsrechnung
.
396 . Inkasso-Gutschrift
.
420 . Optical Character Reading (OCR) payment credit note
.
456 . Belastungsanzeige
.
457 . Storno einer Belastung.
.
458 . Storno einer Gutschrift
.
527 . Self billed debit note
.
532 . Gutschrift des Spediteurs
.
553 . Forwarder’s invoice discrepancy report
.
575 . Rechnung des Versicherers
.
623 . Speditionsrechnung
.
633 . Hafenkostenrechnung

751 . Invoice information for accounting purposes
.
780 . Frachtrechnung
.
817 . Claim notification
.
870 . Konsulatsfaktura
.
875 . Partial construction invoice
.
876 . Partial final construction invoice
.
877 . Final construction invoice
.
935 . Zollrechnung

**Reply by stephanstapel** [reply]
I decided against throwing in all codes since this renders the component being unusable

**Reply by kane95443** [reply]
All good, thank you for the support. I find it impressive how much work you put into the library. Thank you very much for that.
